dev_docs/tutorials/performance/adding_custom_performance_metrics.mdx

```diff
@@ -404,6 +404,38 @@ An event using custom metrics will be indexed with the following structure:
 }
 ```
 
+#### Add inline documentation
+
+The description metadata field is allowing teams to document how a page is instrumentented
```

Comment:
Shall we also mention the limit and ask the teams to add a short description + readme to expand further?

Reply:
The topic that we discussed about using the description field in a README for easier discoverability and expand the explanation with images and so on.